tests/wpt/metadata-css/css-transforms-1_dev/html/transform3d-sorting-004.htm.ini

```diff
@@ -0,0 +1,3 @@
+[transform3d-sorting-004.htm]
+  type: reftest
+  disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1362543
```

Comment:
Do we really need to disable these?

We can just mark them as failing with a link to that bug.